examples/graceful-shutdown/graceful-shutdown/server.go

```diff
@@ -44,5 +44,10 @@ func main() {
 	if err := srv.Shutdown(ctx); err != nil {
 		log.Fatal("Server Shutdown:", err)
 	}
+	// catching ctx.Done(). timeout of 5 seconds.
```

Comment:
you also should update readme example, here: https://github.com/gin-gonic/gin#graceful-restart-or-stop
because ci run `make embedmd` failed.